### PR TITLE
Added: unload or close action for tabMiddleClick

### PR DESCRIPTION
**Pull request by JarKz**
# Summary

Adds the close action for unloaded tabs or the unload for active tabs on middle click at the same time. I think it's very useful for some people.

Affected options:
- middle click
- middle click modifier (ctrl + shift)
- middle click close button

### CODE REVIEW

**Comment by webHikari** [comment]
ㅤㅤ ㅤㅤㅤㅤㅤㅤ

File: src/sidebar/components/tab.vue
Comment (webHikari) [inline src/sidebar/components/tab.vue:1]: Switch-case?

**Review by webHikari — APPROVED** [review]

File: src/sidebar/components/tab.vue
Comment (JarKz) [reply src/sidebar/components/tab.vue:1]: Nope, it's not about refactor code, but adding new feature. So I leave the if-statements as was with adding new one.

**Comment by AlekseyCorsello** [comment]
🤫🧏‍♂️

File: src/sidebar/components/tab.vue
Comment (webHikari) [reply src/sidebar/components/tab.vue:1]: Refactor it, author's last commit was 2 months ago, he doesn't give a fuck anyway

**Comment by webHikari** [comment]
https://github.com/mbnuqw/sidebery/assets/158005693/6e98b239-b840-48b2-863f-653b7f4e6893

**Comment by mbnuqw** [comment]
Thanks

**Pull request merged**